sdk/aqueduct/utils.py

```diff
@@ -222,7 +222,10 @@ def serialize_function(
         # Write function source code to file
         source_file = "{}.py".format(op_name)
         with open(os.path.join(dir_path, source_file), "w") as f:
-            source = inspect.getsource(func)
+            try:
+                source = inspect.getsource(func)
+            except Exception as e:
+                source = f"We cannot get source file due to:\n{str(e)}"
```

Comment:
It will be something like 'OSError: could not get source code'. I do want to have some description like 'it's likely you are running in a python interpreter', but I didn't come up with more precise description here

Reply:
welcome to suggestions!

Reply:
Hm, I'm not sure I understand what's happening here. Are we just setting a dummy source file? What is the source file used for?

Reply:
It's for displaying content in UI. The actual execution uses the pickled object a few lines above

Reply:
Got it. Maybe just say `unknown` for now?

Reply:
you mean the content of this file?

Reply:
yep